src/middleware/auth.ts

```diff
@@ -26,7 +26,7 @@ const authMiddleWare = async (
       res.status(401).send({ message: 'Invalid token: User not found' });
       return;
     }
-    req.user = user;
+    req.user = { id: user.id };
     next();
   } catch (error: any) {
```

Comment:
This lint error will be solved with `error: unknown` because TS don't know the error type in a `catch` block.
[example:unknown-in-catch](https://www.typescriptlang.org/play/?useUnknownInCatchVariables=true&ts=4.4.2#code/PTAECEFMGMEMFcDOlQClYDdYGVoCcBLABwBdRYAbCgewHdFQSALPOggOwHNz2BPULBXiQANKAAqvIpFyFSAKBCgAJtUgN21MonhEi1PGWUwKsQl0ZMUJKSmoAzHqEh5WeefJJ5+Ab3mhQJQA6IPkAX1A4EmgmUAAKSABKUB9QMI8lAAkCRBIDAjgqXjFmHNAmWAYAW0hYdjJmWAarUAADSFbQDksUKJjFMFpqeAplFUh7BAoG6h5eIIkmMsoaWkgx+wMe0Hs8SHXqKsZqAfI8ACMCLzMCCn5YaGh1Bjr+IlZpQ3nQAHUr2IALEEAAxiNYAcgwKBo1GQ7HWp2YKD2uUI0BIBGo7AYWMYtnIiEQBE47Bq9S67G2fVi0FMSEgiNmKzooHOWlirVenTqY1a8HYAGtNLR2K1QqdsLAaqzIBUMJi8KBaP85gAuTzeFL+QJgELhSJNGLxJJagKQIK5B4CgDc4QyYAAogAPIgUApXGVyhVKlX8oV0djqjW+bXBUIRanG1WgP3C9jJPwBJQATWGBvYmjI9LaHXIZBWMfqt22NmkobAiF4uUgRwotTw2KVFTIHpyYk2iqqBhQHE2oGQkHVSbAkCdUtdg9jAYAtDzp-Coe4zRaSFbbdqCI4EhTLewng5QNhePVYE6Ha4DAntcvLdAbdr0mEgA)